src/pybind/mgr/volumes/fs/nfs.py

```diff
@@ -8,12 +8,149 @@
 import orchestrator
 from .fs_util import create_pool
 
+from typing import List
+
 log = logging.getLogger(__name__)
 
+def available_clusters(mgr):
```

Comment:
Comment that you're using cephadm to inform what clusters exist. Note, you're not looking at all of the objects in the RADOS pool/namespace to see what clusters have previously been setup.

At some point (not today), we need to reconcile what cephadm says exists and what the nfs clustering module thinks should exist.

Reply:
> Comment that you're using cephadm to inform what clusters exist. Note, you're not looking at all of the objects in the RADOS pool/namespace to see what clusters have previously been setup.

Done.
> 
> At some point (not today), we need to reconcile what cephadm says exists and what the nfs clustering module thinks should exist.

I have added todo note for it.